src/xdp.props

```diff
@@ -22,9 +22,8 @@
   </PropertyGroup>
   <ItemGroup>
     <!-- eBPF currently has separate packages for each architecture. -->
-    <PackageReference Include="eBPF-for-Windows" Version="0.18.0" GeneratePathProperty="true" Condition="'$(ImportEbpf)' == 'true' AND '$(Platform)' == 'x64'"/>
-    <PackageReference Include="eBPF-for-Windows.arm64" Version="0.20.0" GeneratePathProperty="true" Condition="'$(ImportEbpf)' == 'true' AND '$(Platform)' != 'x64'"/>
-    <PackageReference Include="eBPF-for-Windows.x64" Version="0.20.0" GeneratePathProperty="true" Condition="'$(ImportEbpf)' == 'true' AND '$(Platform)' != 'x64'"/>
+    <PackageReference Include="eBPF-for-Windows.arm64" Version="0.21.0" GeneratePathProperty="true" Condition="'$(ImportEbpf)' == 'true' AND '$(Platform)' != 'x64'"/>
```

Comment:
i forget why this was written as `'$(Platform)' != 'x64'` but can we see if this PR works with `'$(Platform)' == 'arm64'` ?

Reply:
ah, ok, i think this will need a bit more of a rewrite. here's what i suggest:

add an `EbpfVersion` property in the `PropertyGroup` above, set to 0.21.0

add here:

```
    <PackageReference Include="eBPF-for-Windows.$(Platform)" Version="$(EbpfVersion)" GeneratePathProperty="true" Condition="'$(ImportEbpf)' == 'true''"/>
    <PackageReference Include="eBPF-for-Windows.$(HostPlatform)" Version="$(EbpfVersion)" GeneratePathProperty="true" Condition="'$(ImportEbpf)' == 'true' AND '$(Platform)' != '$(HostPlatform)''"/>
```